shell/platform/glfw/client_wrapper/include/flutter/flutter_window.h

```diff
@@ -88,6 +88,13 @@ class FlutterWindow {
     FlutterDesktopWindowSetPixelRatioOverride(window_, pixel_ratio);
   }
 
+  // Sets the min/max size of |flutter_window| in screen coordinates. Use
+  // kFlutterDesktopDontCare for any dimension you wish to leave unconstrained.
+  void SetSizeLimits(FlutterDesktopSize minimum_size,
+                     FlutterDesktopSize maximum_size) {
```

Comment:
We may want to use different structs/constants rather than relying on the wrapped API's types, but that's fine to revisit later.